YAML/python_bot.yaml

```diff
@@ -0,0 +1,45 @@
+## ID - auto generated or manual alpha numeric, must be unique across RLC
+id: python_bot
+
+## Name - human readable name (e.g., Install MongoDB, Change JIRA password...)
+name: python bot
```

Comment:
Since the name is visible to the end user it is best to give something that clearly conveys the intent. So "Test BOT Using Python Script" is a better name. Also please capitalize the starting letter of each word.